### ISSUE TITLE
Getting a lot of "constant ::NIL is deprecated"

### ISSUE THREAD

**Issue by maestromac** [cloudinary/cloudinary_gem#329]
I just bumped cloudinary from 1.9.1 to 1.11.0 and I'm getting a lot of the following warning in my test suite.

```
/Users/username/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/cloudinary-1.11.0/lib/cloudinary/utils.rb:1074: warning: constant ::NIL is deprecated
```

Is it because I'm using a deprecated method? I'm not seeing any other warning in my log.

**Comment by tkawa** [comment]
This commit causes that warning.
https://github.com/cloudinary/cloudinary_gem/commit/48c918a498b69e9f7c8093c2854cc8422cb2c15a#diff-6b76752efa34ce443a5f5607f7452c35R1069

**Comment by tocker** [comment]
Fixed and published in version [1.11.1](https://github.com/cloudinary/cloudinary_gem/releases/tag/1.11.1)
